app/src/main/java/com/framgia/forder/screen/mainpage/shop/ItemShopViewModel.java

```diff
@@ -65,10 +71,22 @@ public String getOwnerEmail() {
         return "";
     }
 
-    public String getStatus() {
-        if (mShop != null) {
-            return mShop.getStatus();
+    @Bindable
+    public int getStatusShop() {
+        return mStatusShop;
+    }
+
+    public void setStatusShop(int statusShop) {
+        mStatusShop = statusShop;
+        notifyPropertyChanged(BR.statusShop);
+    }
+
+    private void getImageStatus() {
+        if (Objects.equals(R.string.active, mShop.getStatus())) {
+            setStatusShop(R.drawable.ic_open_shop);
+        }
+        if (Objects.equals(R.string.inactive, mShop.getStatus())) {
```

Comment:
ERROR: [CORRECTNESS][EC_UNRELATED_TYPES] Call to Integer.equals(String) in com.framgia.forder.screen.mainpage.shop.ItemShopViewModel.getImageStatus()
http://findbugs.sourceforge.net/bugDescriptions.html#EC_UNRELATED_TYPES